source/docs/prismic/routable-types.md

```diff
@@ -0,0 +1,220 @@
+---
+id: routable-types
+title: Prismic Routable Types
+---
+
+To make a custom Prismic data type routable these steps are required:
+
+- [Create a field to represent the Url of the custom type](#create-a-field-to-represent-the-url-of-the-custom-type)
+- [Create a GraphQL type to represent the custom type](#create-a-graph-ql-type-to-represent-the-custom-type)
+- [Register the Prismic custom type as a routable type](#register-the-prismic-custom-type-as-a-routable-type)
+- [Map GraphQL type to a component](#map-graph-ql-type-to-a-component)
```

Comment:
as I was saying on Slack, the anchors do not work (mostly case issues) it seems

See https://deploy-preview-208--heuristic-almeida-1a1f35.netlify.app/docs/prismic/routable-types.html